### PR TITLE
Sound

### PR DESCRIPTION
**Pull request by JaspervanTilburg**

### CODE REVIEW

File: src/main/java/doob/controller/HighscoreMenuController.java [hunk 1]
```diff
@@ -69,7 +70,7 @@ public void insertScore(final int score, int player) {
 		final Stage dialog = new Stage();
 		final InputPopup popup = App.popup(dialog,
 				"/FXML/InputPopup.fxml").getController();
-		String text = " Please enter the name of the level:";
+		String text = "Player " + player + " has a highscore! Enter your name:";
 		popup.setText(text);
```
Comment (cbilstra) [inline src/main/java/doob/controller/HighscoreMenuController.java]: Better text indeed

File: src/main/java/doob/util/SoundManager.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package doob.util;
+
+import javafx.scene.media.Media;
+import javafx.scene.media.MediaPlayer;
+
+/**
+ * Class to manage the soundeffects in the game.
+ */
```
Comment (cbilstra) [inline src/main/java/doob/util/SoundManager.java]: I like it that you made it a util class

File: src/main/java/doob/util/SoundManager.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	
+	public static final String DIE_EFFECT = "/Sounds/aww.mp3";
+	public static final String POP_EFFECT = "/Sounds/pop.mp3";
+	public static final String LEVEL_COMPLETE = "/Sounds/congratulations.mp3";
+	public static final String GAME_WON = "/Sounds/applaus.mp3";
+	public static final int MAX_VOLUME = 100;
+	
+	private SoundManager() {
+	}
+
```
Comment (cbilstra) [inline src/main/java/doob/util/SoundManager.java]: constructor can be removed right?

Comment (JaspervanTilburg) [inline src/main/java/doob/util/SoundManager.java]: No than it gives a checkstyle error that a public or default constructor is now allowed.

Comment (cbilstra) [inline src/main/java/doob/util/SoundManager.java]: Alright, let's merge then!

**Pull request merged**